Lib/_pyrepl/simple_interact.py

```diff
@@ -82,17 +82,15 @@ def _clear_screen():
 
 
 def run_multiline_interactive_console(
-    namespace: dict[str, Any],
+    *,
+    console: code.InteractiveConsole,
     future_flags: int = 0,
```

Comment:
If we're going to make `console` required, then it shouldn't be forced as a keyword parameter:
```suggestion
    console: code.InteractiveConsole,
    *,
    future_flags: int = 0,
```
(If you make it optional again, per my other review comments, then ignore this suggestion.)